### DISCUSSION TITLE
Supporting DB2 database

### DISCUSSION THREAD

**Discussion by HenryMiller** [amplication/amplication#2267] [Ideas]
Hello ,
do you have any plan to include DB2 fo IBM I (aka AS400).

**Comment by yuval-hazaz** [comment]
hi @HenryMiller 
Not currently.
Would you mind sharing some details on the use case?

**Comment by mshidlov** [comment]
Hey @HenryMiller,
It will help us understand your use case for considering future support in DB2.

We are currently supporting PostgreSQL.

In our road map, we plan to add support to more databases.

We are generating applications that work with Prisma (next step ORM),
So you can update your Prisma client in the generated app to connect to a different database supported by Prisma.

In the first step, we plan to support Prisma-supported databases in our product.

In the next step, other databases will be considered if there is a demand from the community.

you can find here the list of Prisma currently supported databases ->
https://www.prisma.io/docs/reference/database-reference/supported-databases

**Comment by mshidlov** [comment]
I'm converting this issue to a discussion until we decide to support DB2
